chrome/content/preferences.xul

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<?xml-stylesheet href="chrome://global/skin/" type="text/css"?>
+
+<prefwindow id="universalsearch-prefs" title="Universal Search Preferences" xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul">
+  <prefpane id="unviersalsearch-pane">
+    <preferences>
+      <preference id="pref_server" name="extensions.universalsearch.server" type="string"/>
+    </preferences>
+    <hbox align="center">
+      <label control="server" value="Server: "/>
+      <textbox preference="pref_server" id="server"/>
+    </hbox>
+  </prefpane>
+</prefwindow>
```

Comment:
After some digging in MDN, I think Inline Options are the way to go: that'll give us the controls visible inside about:addons. https://developer.mozilla.org/en-US/Add-ons/Inline_Options